pkg/imgpkg/cmd/registry_flags.go

```diff
@@ -19,6 +20,8 @@ type RegistryFlags struct {
 	Password string
 	Token    string
 	Anon     bool
+
+	ResponseTimeout time.Duration
```

Comment:
this should be ResponseHeaderTimeout. ResponseTimeout implies that it's a timeout for the entire response which isnt the case of how we are using this value.

Reply:
ok done